pkg/build/registry/build/etcd/etcd.go

```diff
@@ -45,5 +45,10 @@ func NewStorage(s storage.Interface) *REST {
 		Storage:             s,
 	}
 
-	return &REST{store}
+	buildStorage = &REST{store}
+	revisionStore := *store
+	revisionStore.UpdateStrategy = build.RevisionStrategy
```

Comment:
isn't this going to leave revisionStore with a build.Strategy for Create+Delete?  and wouldn't we want revisionStore to reject creates+deletes?